src/docs/guides/development-teams-and-deployment/README.md

```diff
@@ -0,0 +1,31 @@
+# Working on a Site with a Development Team
+
+If multiple people will be working on the same website (development team approach), or if you plan to work on multiple versions of the same website (development / staging / production), then continue reading.
+
+## App_Data
+
+In OrchardCore, the App_Data folder is used to store tenant details for the system `tenants.json`, the log files `\logs` and also details for each tenant in a tenant specific folder `\sites\{tenant name}`.
+
+A tenant folder, by default, stores "Keys", "Media" (uploaded content) and the `app.settings` file. It also stores the definition of each Content Type in the `ContentDefinition.json` file.
+
+It is possible to store the Content Types in the database instead.
+
+## Development Team Approach
+
+Possibly the first choice to make using the development team approach is whether you can share a database with real-time access during development. If so, then it is a good idea to move to using the Database storage approach for Content Types. If not, then leaving them in the file system and sharing them through git commits is likely the easiest way.
```

Comment:
Do you mean that all developers connect to a single DB? People will step on each others' feet with conflicting and work in progress schema changes.

Reply:
Sure. How do you manage DB migrations though? I think some note on it would be necessary here.